site/content/entry/yuzu-progress-report-apr-2020/index.md

```diff
@@ -0,0 +1,208 @@
+[Rodrigo](https://github.com/ReinUsesLisp) fixed most of them by improving the accuracy of the shader decoder. This also fixes some bugs on older Nvidia GPUs too.
+
+{{< single-title-imgs
+    "Monochrome no more (Fire Emblem: Three Houses)"
+    "./29.png"
+    "./30.png"
+  >}}
+
+{{< single-title-imgs
+    "No greasy black peels (Super Monkey Ball: Banana Blitz HD)"
+    "./27.png"
+    "./28.png"
+  >}}
+  
+Speaking of shader decoding accuracy, [Rodrigo](https://github.com/ReinUsesLisp) improved the precision of the `HADD2` and `HMUL2` instructions, fixing some long standing bugs like the fog in `Fire Emblem: Three Houses` and the excessive brigthness in most maps in `Super Smash Bros. Ultimate`.
+
+{{< single-title-imgs
+    "You can now take off your sunglasses (Super Smash Bros. Ultimate)"
+    "./33.png"
+    "./34.png"
+  >}}
+  
+{{< single-title-imgs
+    "From steampunk, to the proper medieval fantasy setting (Fire Emblem: Three Houses)"
+    "./16.png"
+    "./15.png"
+  >}}
+
+We talked about this in its own [article](https://www.patreon.com/posts/35856792), but it deserves a mention here:
+[bunnei](https://github.com/bunnei) rewrote the Virtual Memory Manager (known internally as `Project Epimetheus`), which drastically reduced RAM use by games, and helped Multicore support (also known as`Project Prometheus`) become a reality.
+
+One of the features in yuzu, and a common one in other emulators, is the ability to separate the rendering thread from the main emulation thread. This makes better use of CPU's resources, while also helping deliver better performance.
+That is exactly what Asynchronous GPU does in yuzu. Thanks to [Blinkhawk](https://github.com/FernandoS27), our Emulated GPU improved drastically, achieving better performance, stability, and accuracy.
+These changes introduce the option to adjust the GPU accuracy level while playing.
+
+{{< imgs
+    "./31.png| Performance may be reduced with more accurate levels"
+  >}}
+
+{{< single-title-imgs
+    "Normal accuracy (left) and High accuracy (right) (Super Mario Odyssey)"
+    "./v1.mp4"
+    "./v2.mp4"
+  >}}
+
+
+## One Line to Fix 'em All
+
+`Toki Tori` had sound problems that were related to a floating point instruction, and while investigating the issue, 
+[Merry](https://github.com/MerryMage) and [bunnei](https://github.com/bunnei) were worried that it may be a problem in [Dynarmic](https://github.com/MerryMage/dynarmic), our ARM to x64 recompiler.
+Turns out the problem was just a bad default value in yuzu’s Dynarmic configuration.
+Remember what we said about bunnei and serendipity? With a simple one line change, we fixed tons of bugs on a lot of games! Special mention goes to `Pokémon Sword/Shield`, where thanks to this single line change, the game no longer softlocked and became fully playable!
```

Comment:
You forgot to change:

`I'd change With a simple one line change, we fixed tons of bugs on a lot of games! Special mention goes to `Pokémon Sword/Shield`, where thanks to this single line change, the game no longer softlocked and became fully playable!`
to
`With a simple one line change, a plethora of bugs were resolved! A special mention goes to `Pokémon Sword/Shield`, as that single line fixed the softlock that plagued us for months and made the game fully playable!`